cscs-checks/apps/gromacs/gromacs_check.py

```diff
@@ -129,6 +129,7 @@ def __init__(self):
         }
 
 
+# FIXME: This test is obsolete; it is kept only for reference.
```

Comment:
should we comment the valid systems in this file?

Reply:
I was thinking that these tests will be replaced by your versions, so, I've just put a `FIXME` tag.